src/App.vue

```diff
@@ -750,6 +750,7 @@ export default {
 		:deep(.app-navigation-toggle) {
 			/* Force white handle when inside a call */
 			color: #D8D8D8;
+			background-color: transparent !important;
```

Comment:
> Not sure if we need to apply dark-theme here either

We manually change all buttons in the call view to dark theme because the call view is always dark. Having it white seems very bright and eye-catching. Why not always use a dark theme here?

Reply:
cc @marcoambrosini for a design change review

Reply:
Also agree that we should keep the buttons dark themed in the call view. It should look no different from other buttons in terms of styling.

Reply:
But other buttons are not dark-themed... they are transparent with semi-ttamsparent hover effect, which won't fit for the toggle overlapping them 🤕

Reply:
But the light theme looks even more eye-catching on a dark background around other dark buttons.

I'm fine with some other colors, but the transparent button overlaps on the mobile view.